webrtc.html

```diff
@@ -3484,7 +3484,13 @@
                   prior to the createOffer() call.
                 </p>
                 <p>
-                  In all other situations, it will be disregarded.
+                  Whenever this is given a false value, and the
+                  <code><a>RTCPeerConnection</a></code> has a non-stopped
+                  "sendrecv" audio transceiver, createOffer() MUST as its first
```

Comment:
Sorry to be pedantic, but what if there are multiple such transceivers? Does this affect all of them, just the first, just the last?